mev_inspect/sandwiches.py

```diff
@@ -42,32 +42,57 @@ def _get_sandwich_starting_with_swap(
     ]:
         return None
 
-    for other_swap in rest_swaps:
-        if other_swap.transaction_hash == front_swap.transaction_hash:
+    for back_swap in rest_swaps:
+        if back_swap.transaction_hash == front_swap.transaction_hash:
             continue
 
-        if other_swap.contract_address == front_swap.contract_address:
+        if back_swap.contract_address == front_swap.contract_address:
             if (
-                other_swap.token_in_address == front_swap.token_in_address
-                and other_swap.token_out_address == front_swap.token_out_address
-                and other_swap.from_address != sandwicher_address
+                back_swap.token_in_address == front_swap.token_in_address
+                and back_swap.token_out_address == front_swap.token_out_address
+                and back_swap.from_address != sandwicher_address
             ):
-                sandwiched_swaps.append(other_swap)
+                sandwiched_swaps.append(back_swap)
             elif (
-                other_swap.token_out_address == front_swap.token_in_address
-                and other_swap.token_in_address == front_swap.token_out_address
-                and other_swap.from_address == sandwicher_address
+                back_swap.token_out_address == front_swap.token_in_address
+                and back_swap.token_in_address == front_swap.token_out_address
+                and back_swap.from_address == sandwicher_address
             ):
                 if len(sandwiched_swaps) > 0:
+                    profit_amount: float
```

Comment:
I think this formula needs some documentation in the code. Explain a little what is happening in here, and if you can put a math formula in clearer syntax than python code that would be nice.

@gheise how do we know if the formula is correct?

Reply:
thanks for pointing that out, the reasons for this are mathematically expressible. I will add further detail on the PR about why i chose this formula and what it means, but i think we should leave this out of the code